core/audits/byte-efficiency/legacy-javascript.js

```diff
@@ -177,7 +177,12 @@ class LegacyJavascript extends ByteEfficiencyAudit {
   }
 
   static getPolyfillData() {
-    return [
+    /** @type {Array<{name: string, modules: string[], corejs?: true}>} */
```

Comment:
nit: I see we never do `corejs: false` so technically the type is correct, but why not just make the type `corejs?: boolean` in case we want to in the future?